graph-manager-docs/source/setup-analytics.md

```diff
@@ -33,47 +32,38 @@ server.listen().then(({ url }) => {
 });
 ```
 
-For advanced configuration options to set up logging, error filtering, and client-aware metrics reporting take a look at our [server documentation](https://www.apollographql.com/docs/apollo-server/features/metrics.html).
-
-## Other servers
+Alternatively, you can assign your API key to the `ENGINE_API_KEY` environment variable in the environment where Apollo Server will run.
```

Comment:
this language seems backwards to me. I think you would assign the `ENGINE_API_KEY` environment variable to your api key (e.g. `service:foo:193401u5j93`), not the other way around.